requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.23.0
```

Comment:
We also require `boto3`, no?

Reply:
No need to add. All lambda functions already have `boto3` library by default. If we add `boto3` in the virtualenv it would increase the size of the deployment package with no difference.

Reply:
Got it. But we need to manually install `boto3` for testing locally? Or how's it tested?

Reply:
That's another thing. The best option to test is to create a local runner or a dev/test lambda function.